requirements.txt

```diff
@@ -1,7 +1,9 @@
 autopep8
+python-dbusmock
 jeepney
 lyricwikia
 pycodestyle
 pylint
+pytest
```

Comment:
I added 2 packages -- not sure why I added them in these specific lines, weird.  But anyway, `pytest` a pretty modern test finder and runner, so that should be fine.  `python-dbusmock` is a convenient library to build mockup testing for libraries that depend heavily on dbus communication.